<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>litescreengui</class>
 <widget class="QMainWindow" name="litescreengui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LiteScreen</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Client</string>
       </attribute>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>15</x>
          <y>76</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Connect To</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="to_server_ip">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>80</y>
          <width>221</width>
          <height>31</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>IP Address To Connect</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="to_server_port">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>80</y>
          <width>113</width>
          <height>31</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Port</string>
        </property>
       </widget>
       <widget class="QPushButton" name="client_start">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>220</y>
          <width>161</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Start Screen Sharing</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Server</string>
       </attribute>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>40</y>
          <width>201</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>IP Address Of the System</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="txt_server_ip">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>40</y>
          <width>241</width>
          <height>31</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="txt_server_port">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>100</y>
          <width>113</width>
          <height>27</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>100</y>
          <width>111</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Port Address</string>
        </property>
       </widget>
       <widget class="QPushButton" name="server_start">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>220</y>
          <width>191</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Start Sharing</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>742</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
